src/mappings/DAORegistry/mapping.ts

```diff
@@ -3,11 +3,20 @@ import 'allocator/arena';
 import { Address, BigInt, store } from '@graphprotocol/graph-ts';
 
 // Import event types from the Token contract ABI
-import { DAORegistry, Propose, Register, UnRegister } from '../../types/DAORegistry/DAORegistry';
+import { Propose, Register, UnRegister } from '../../types/DAORegistry/DAORegistry';
+
+import { Avatar } from '../../types/DAORegistry/templates';
 
 import * as domain from '../../domain';
 
 export function handlePropose(event: Propose): void {
+  // Start tracking the new avatar template if we aren't already
```

Comment:
the purpose of daoRegistery is to index new daos in the dapp(alchemy) layer.
I guess the daotracker(according to the pr description) is the one the subgraph should follow in order to index a new dao .

Reply:
Yes I agree, the reason this was added was to show that the feature was working. This was requested further up in the PR, along with corresponding tests, to validate datasource templates was working. It's essentially stub code to show the functionality. I'm in favor of removing this.